extension/src/constants/featureFlag.ts

```diff
@@ -0,0 +1,3 @@
+declare const EXPERIMENTAL: string;
+const _EXPERIMENTAL = EXPERIMENTAL;
+export { _EXPERIMENTAL as EXPERIMENTAL };
```

Comment:
now you can import EXPERIMENTAL as a constant and show/hide features based off its value

Reply:
My experience using env like this is that it means the code can't be 100% stripped from output when it's irrelevant, cuz dead code elimination doesn't trace imports for constant values. `DefinePlugin` overwrites the exact string variable name with the exact string output, if it's then used as an export then that value is still interpreted at runtime

I think that's probably fine, but something to be aware of

Reply:
Interesting - this only being done to deal with some annoying TS issue. This can be changed if we run into issues